site/frontend/src/pages/compare/compile/compile-page.vue

```diff
@@ -222,6 +264,17 @@ const filteredSummary = computed(() => computeSummary(comparisons.value));
     :selected-metric="selector.stat"
     :metrics="benchmarkInfo.compile_metrics"
   />
+  <div
+    v-if="comparesIdenticalCommits"
```

Comment:
I think this should look at the other property, like

```suggestion
    v-if="selfCompareBackend"
```

Otherwise, we'll show this piece of UI even if there isn't cranelift data for a commit: which in practice will always happen on master + perf.rlo.

Reply:
Using this property would make it recursive :sweat_smile: I introduced another one.